redirector/webhook_receiver.py

```diff
@@ -1,163 +1,88 @@
-            # Deal with the exception here, and/or rethrow at your discretion.
-            raise e
-        elif e.response["Error"]["Code"] == "InternalServiceErrorException":
-            # An error occurred on the server side.
-            # Deal with the exception here, and/or rethrow at your discretion.
-            raise e
-        elif e.response["Error"]["Code"] == "InvalidParameterException":
-            # You provided an invalid value for a parameter.
-            # Deal with the exception here, and/or rethrow at your discretion.
-            raise e
-        elif e.response["Error"]["Code"] == "InvalidRequestException":
-            # You provided a parameter value that is not valid for the current state of the resource.
-            # Deal with the exception here, and/or rethrow at your discretion.
-            raise e
-        elif e.response["Error"]["Code"] == "ResourceNotFoundException":
-            # We can't find the resource that you asked for.
-            # Deal with the exception here, and/or rethrow at your discretion.
-            raise e
-    else:
-        # Decrypts secret using the associated KMS CMK.
-        # Depending on whether the secret is a string or binary, one of these fields will be populated.
-        if "SecretString" in get_secret_value_response:
-            secret = get_secret_value_response["SecretString"]
-            return json.loads(secret)
-        else:
-            logger.error("binary secret not implemented")
-            # raise NotImplementedError
-            # decoded_binary_secret = base64.b64decode(get_secret_value_response['SecretBinary'])
-            # return decoded_binary_secret
-
-
-def get_box_client(app_user=True):
-    from boxsdk import Client, JWTAuth
-
-    secret = get_secret()
-
-    CLIENT_ID = secret["clientID"]
-    CLIENT_SECRET = secret["clientSecret"]
-    PUBLIC_KEY_ID = secret["publicKeyID"]
-    PRIVATE_KEY = secret["privateKey"].replace("\\n", "\n")
-    PASSPHRASE = secret["passphrase"]
-    ENTERPRISE_ID = secret["enterpriseID"]
-    WEBHOOK_KEY = secret["webhook_key"]
-
-    auth = JWTAuth(
-        CLIENT_ID,
-        CLIENT_SECRET,
-        ENTERPRISE_ID,
-        PUBLIC_KEY_ID,
-        rsa_private_key_data=PRIVATE_KEY,
-        rsa_private_key_passphrase=PASSPHRASE,
-    )
-    auth.authenticate_instance()
+import common
 
-    client = Client(auth)
 
-    if app_user:
-        users = client.users()
-        try:
-            appuser = users.next()
-        except StopIteration:
-            logger.info(
-                "no app user exists, so the service account will be used as the box api client"
-            )
-            return client
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
 
-        appClient = client.as_user(appuser)
+HANDLED_FILE_TRIGGERS = {
+    "SHARED_LINK.CREATED",
+    "SHARED_LINK.UPDATED",
+    "SHARED_LINK.DELETED",
+    "FILE.TRASHED",
```

Comment:
Added these, it seems that we don't get any SHARED_LINK events when the file is trashed, but the download links stop working.